core/src/main/java/org/frankframework/util/MessageUtils.java

```diff
@@ -68,7 +68,7 @@ public class MessageUtils {
 
 	public static final String JSON_TEMPLATE_VALUE_QUOTED = "{\"%s\": \"%s\"}";
 	public static final String JSON_TEMPLATE_VALUE_UNQUOTED = "{\"%s\": %s}";
-	public static final String GENERATED_MESSAGE_ID_PREFIX = "synthetic-message-id-";
+	public static final String FALLBACK_MESSAGE_ID_PREFIX = "fallback-message-id-";
```

Comment:
Ik weet niet hoe handig het is deze aan te passen, ik vrees dat dit hard in de larva scenarios staat en niet makkelijk geignored kan worden.

Reply:
Ik dacht dat die message IDs sowieso al genegeerd zouden worden omdat ze altijd anders zijn. We zullen zien. Als het breekt, draait ik het terug.

Reply:
Het lijkt geen probleem te zijn in de Larva scenarios maar ik ga hem toch anders noemen. "Fallback" klinkt me toch te negatief, alsof iemand vergeten was een msg-id mee te sturen, terwijl het eigenlijk een normale situatie is in geval van het aanroepen van een sub-adapter.